phel-config.php

```diff
@@ -2,17 +2,16 @@
 
 declare(strict_types=1);
 
-return [
-    'src-dirs' => ['src/phel'],
-    'test-dirs' => ['tests/phel'],
-    'vendor-dir' => 'vendor',
-    'export' => [
-        'directories' => ['src/phel'],
-        'namespace-prefix' => 'PhelGenerated',
-        'target-directory' => 'src/PhelGenerated',
-    ],
-    'out-dir' => 'out',
-    'ignore-when-building' => ['src/phel/local.phel'],
-    'keep-generated-temp-files' => false,
-    'format-dirs' => ['src', 'tests'],
-];
+return (new \Phel\Config\PhelConfig())
+    ->setSrcDirs(['src/phel'])
```

Comment:
Do we want to use the suffix `set` for setters? @Chemaclass @jenshaase

Reply:
I think it makes sense to have setters here, I think it makes the fluent interface more natural.